packages/validators-util/src/isValidRegistreringsnummer/isValidRegistreringsnummer.test.ts

```diff
@@ -2,10 +2,34 @@ import { isValidRegistreringsnummer } from "./isValidRegistreringsnummer";
 
 describe("isValidRegistreringsnummer", () => {
     it("should return true for two letters and 5-digit numbers", () => {
-        expect(isValidRegistreringsnummer("AB12345")).toBe(true);
+        expect(isValidRegistreringsnummer({ value: "AB12345" })).toBe(true);
     });
 
     it("should return false if only one letter and more than 5-digit numbers", () => {
-        expect(isValidRegistreringsnummer("A1234567")).toBe(false);
+        expect(isValidRegistreringsnummer({ value: "A1234567" })).toBe(false);
+    });
+
+    it("should return false for two letters and 4-digit numbers", () => {
+        expect(isValidRegistreringsnummer({ value: "AB1234" })).toBe(false);
+    });
+
+    it("should return true if product=bil and value two letters and 5-digit numbers", () => {
+        expect(isValidRegistreringsnummer({ value: "AB12345", product: "bil" })).toBe(true);
```

Comment:
`product` er kanskje et litt for team-spesifikt begrep i denne sammenhengen. Hva med `vehicle`, `type`, `class`, `category`?